packages/configs/package.json

```diff
@@ -20,9 +20,6 @@
 		"@typescript-eslint/parser": "^5.49.0"
 	},
 	"prettier": "./prettier.config.js",
```

Comment:
Because we can't export the prettier config from the same package as the ESLint config (or if we can it's confusing because of the name of the package), let's move the prettier config to its own package.